static/js/muting_ui.js

```diff
@@ -16,6 +16,46 @@ exports.rerender = function () {
     }
 };
 
+exports.mute_notification = (function () {
+    var timeout,
+        animate = {
+            fadeOut: function ($mute) {
+                $mute.fadeOut(500).removeClass("show");
+            },
+            fadeIn: function ($mute) {
+                $mute.fadeIn(500).addClass("show");
+                clearTimeout(timeout);
+            }
+        },
```

Comment:
I think most of our code uses one variable per var statement.  It just makes it a bit easier when you're scanning the code to know that something's not a closure variable.